mkFit/mkFit.cc

```diff
@@ -341,6 +341,10 @@ void test_standard()
 
         StdSeq::LoadHits(ev, eoh);
 
+	std::vector<DeadVec> deadvectors(ev.layerHits_.size());
+#include "deadmodules.h"
+	StdSeq::LoadDeads(eoh, deadvectors);
```

Comment:
We could in principle call this just once where EventOfHits objects are created, one for each event of flight.
[ or even share `vecvecPhiBinDead_t   m_phi_bin_deads` via a const ptr ... this has the added benefit that if the ptr is null (not set), we know dead regions are not being used ]

Reply:
Forgot to say, Slava says pixel dead regions change more frequently so for those we still need a per instance / per event update -- but this probably only applies to CMSSW implementation.